npm setup failed to compile

**Replication Steps**
1. git clone the project
2. run $ npm run setup --silent
3. Resulted in: Failed to compile
Module not found: Error: Can't resolve '@babel/runtime/helpers/esm/inheritsLoose' in '{MY-FOLDER}/github.com/kentcdodds/advanced-react-patterns-v2/node_modules/react-router-dom/esm'

I have checked that there is file under ..node_modules/@babel/runtime/helpers/esm/inheritsLoose.js

**Specification**
1. npm version 6.9.0

Kindly help to advise on how to make it successfully compiled, Thanks !

--- a/package.json
+++ b/package.json
@@ -23,8 +23,8 @@
     "inquirer": "^5.2.0",
     "is-ci": "^1.1.0",
     "node": "^8.11.1",
-    "npm-run-all": "^4.1.2",
-    "react-scripts": "2.0.0-next.b2fd8db8",
+    "npm-run-all": "^4.1.3",
+    "react-scripts": "2.1.8",
     "replace-in-file": "^3.4.0"
   },
   "scripts": {

